### PR TITLE
Corrected Caput 1, resubmitting

### PR DESCRIPTION
**Pull request by a1wilson**
Thanks for the suggested corrections
- I removed all of the n="P"

### CODE REVIEW

File: lca8s8-d1e52/cod-dlc9qq_lca8s8-d1e52.xml [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+  </teiHeader>
+  <text xml:lang="la">
+    <front>
+      <div xml:id="starts-on">
+        <pb ed="#P" n="11-r"/>
+      </div>
+    </front>
+    <body>
+      <div xml:id="lca8s8-d1e52">
+		<head id="P">Caput 1 [Bnf 12926 Transcription]</head>
```
Comment (jeffreycwitt) [inline lca8s8-d1e52/cod-dlc9qq_lca8s8-d1e52.xml]:
```suggestion
		<head>Caput 1 [Bnf 12926 Transcription]</head>
```

Comment (jeffreycwitt) [reply lca8s8-d1e52/cod-dlc9qq_lca8s8-d1e52.xml]: small schema error that needs to be fixed.

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]

File: lca8s8-d1e52/cod-dlc9qq_lca8s8-d1e52.xml (hunk 1 above)
Comment (a1wilson) [reply lca8s8-d1e52/cod-dlc9qq_lca8s8-d1e52.xml]: Got it. I changed this in my template for future chapters.

**Review by jeffreycwitt — APPROVED** [review]
Looks great! I'm merging now.

**Pull request merged**